crates/openvino/src/node.rs

```diff
@@ -13,6 +13,7 @@ pub struct Node {
 
 impl Node {
     /// Create a new [`Port`] from [`ov_output_const_port_t`].
+    #[inline]
     pub(crate) fn new(ptr: *mut ov_output_const_port_t) -> Self {
```

Comment:
```suggestion
    pub(crate) fn from_ptr(ptr: *mut ov_output_const_port_t) -> Self {
```

For convention sake?